lms/templates/user_dropdown.html

```diff
@@ -38,18 +36,18 @@
 % else:
     <ol class="user">
         <li class="primary">
-            <a href="${reverse('dashboard')}" class="user-link">
-                <span class="sr">${_("Dashboard for:")}</span>
+            <a href="${reverse('learner_profile', kwargs={'username': user.username})}" class="user-link">
+                <span class="sr">${_("Profile for:")}</span>
                 <%
                 username = user.username
                 profile_image_url = get_profile_image_urls_for_user(user)['medium']
                 %>
-                <img class="user-image-frame" src="${profile_image_url}" alt="${_('Profile image for {username}').format(username=username)}">
+                <img class="user-image-frame" src="${profile_image_url}" alt="">
                 <div class="label-username">${username}</div>
             </a>
         </li>
         <li class="primary">
-            <button class="dropdown" aria-haspopup="true" aria-expanded="false"><span class="sr">${_("More options dropdown")}</span><span class="fa fa-sort-desc" aria-hidden="true"></span></button>
+            <button class="dropdown" aria-haspopup="true" aria-expanded="false" title="Account settings menu"><span class="fa fa-sort-desc" aria-hidden="true"></span></button>
             <ul class="dropdown-menu" aria-label="More Options" role="menu">
```

Comment:
@tasawernawaz and this aria-label needs to be updated to match: "Account settings"